# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/blindcoder/dvdstyler/hotfixes.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/Config.h.orig	2005-02-25 20:01:13.323362352 +0100
+++ ./src/Config.h	2005-02-25 20:05:58.288041136 +0100
@@ -67,7 +67,7 @@
 
 #define CONFIG_PROP_T(name, cfgName, defName, cfgType)\
   cfgType Get##name(bool def = false)\
-  { return def ? defName : cfg->Read(cfgName, defName); }\
+  { return def ? defName : cfg->Read(cfgName, (cfgType&)defName); }\
   void Set##name(cfgType value)\
   {\
     wxLogNull log;\
--- ./wxVillaLib/imagjpg.cpp.orig	2005-02-25 19:45:01.000000000 +0100
+++ ./wxVillaLib/imagjpg.cpp	2005-02-25 20:08:56.000024808 +0100
@@ -115,7 +115,7 @@
 
 #define OUTPUT_BUF_SIZE  4096    /* choose an efficiently fwrite'able size */
 
-#if !wxCHECK_VERSION(2,5,0)
+// #if !wxCHECK_VERSION(2,5,0)
 #define wx_init_source my_init_sources
 #define wx_fill_input_buffer my_fill_input_buffer
 #define wx_skip_input_data my_skip_input_data
@@ -124,7 +124,7 @@
 #define wx_init_destination init_destination
 #define wx_empty_output_buffer empty_output_buffer
 #define wx_term_destination term_destination
-#endif
+// #endif
 
 
 #ifdef JPEG_METHOD_LINKAGE
